home/management/commands/s3url.py

```diff
@@ -0,0 +1,47 @@
+import boto3
+from django.core.management.base import BaseCommand
+from home.models import MissionArea
+from UnoCPI.settings import AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY, AWS_STORAGE_BUCKET_NAME
+
+
+# APP_ENV = os.environ.get('APP_ENV')
+
+# function to fix S3 URLs for objects based on env
+def s3UrlPicker():
+    url_list = []
+    name_list = []
+    s3 = boto3.client('s3', aws_access_key_id=AWS_ACCESS_KEY_ID, aws_secret_access_key=AWS_SECRET_ACCESS_KEY)
```

Comment:
Replaced shell script with custom django command in this script to pick object URL directly from S3 bucket